pages/portfolio.html

```diff
@@ -4,9 +4,55 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
+  <link href="../styles/portfolio.css" rel="stylesheet"/>
+  <title>Portfolio</title>
 </head>
 <body>
-  
+  <header>
+    <img src="/assets/in-the-snow.jpg" alt="A woman in the snow" />
+    <h1>Danielle Becker</h1>
+    <nav>
+      <ul>
+        <li><a class="header" href="index.html">Home</a></li>
+        <li><a class="header" href="about.html">About</a></li>
+        <li><a class="header" href="portfolio.html">Portfolio</a></li>                   
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section>
+      <h2>Swap Meet</h2>
+      <a href="https://github.com/wreckbeck/swap-meet">
+        <img src="/assets/farmers-market.jpg" alt="A farmers-market" />
+      </a>
+      <h3>Python</h3>
+      <p>Working with classes and models to defne how vendors swap items. Click on the image to view the project repo.</p>
+    </section>
```

Comment:
Nice! Another way to write the HTML for the portfolio page would be to have an unordered list and each list item could contain your section element